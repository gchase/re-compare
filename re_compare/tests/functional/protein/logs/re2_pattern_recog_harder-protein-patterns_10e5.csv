regex,matchtype,matchnumber,time(ms),span
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),first_match,0,0.5850791931152344,"[-1, -1]"
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),all_matches,0,0.5850791931152344,"[-1, -1]"
DHKNLD.D,first_match,0,0.11515617370605469,"[-1, -1]"
DHKNLD.D,all_matches,0,0.11515617370605469,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,first_match,1,0.5478858947753906,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,1,0.5478858947753906,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,2,0.011682510375976562,"[99268, 99274]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,all_matches,2,0.5595684051513672,"[-1, -1]"
(GI)(G|R)P.Y..K..R,first_match,0,0.16570091247558594,"[-1, -1]"
(GI)(G|R)P.Y..K..R,all_matches,0,0.16570091247558594,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),first_match,0,0.8270740509033203,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),all_matches,0,0.8270740509033203,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),first_match,0,0.6871223449707031,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),all_matches,0,0.6871223449707031,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,first_match,0,0.576019287109375,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,all_matches,0,0.576019287109375,"[-1, -1]"
P..L...KWE.C,first_match,0,0.2586841583251953,"[-1, -1]"
P..L...KWE.C,all_matches,0,0.2586841583251953,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),first_match,0,0.5457401275634766,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),all_matches,0,0.5457401275634766,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,first_match,0,0.1690387725830078,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,all_matches,0,0.1690387725830078,"[-1, -1]"
