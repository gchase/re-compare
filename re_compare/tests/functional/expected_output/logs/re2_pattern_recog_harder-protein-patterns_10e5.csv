regex,matchtype,matchnumber,time(ms),span
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),first_match,0,0.41794776916503906,"[-1, -1]"
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),all_matches,0,0.41794776916503906,"[-1, -1]"
DHKNLD.D,first_match,0,0.2281665802001953,"[-1, -1]"
DHKNLD.D,all_matches,0,0.2281665802001953,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,first_match,1,0.44798851013183594,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,1,0.44798851013183594,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,2,0.032901763916015625,"[99268, 99274]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,all_matches,2,0.48089027404785156,"[-1, -1]"
(GI)(G|R)P.Y..K..R,first_match,0,0.24175643920898438,"[-1, -1]"
(GI)(G|R)P.Y..K..R,all_matches,0,0.24175643920898438,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),first_match,0,0.72479248046875,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),all_matches,0,0.72479248046875,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),first_match,0,0.5681514739990234,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),all_matches,0,0.5681514739990234,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,first_match,0,0.4239082336425781,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,all_matches,0,0.4239082336425781,"[-1, -1]"
P..L...KWE.C,first_match,0,0.20003318786621094,"[-1, -1]"
P..L...KWE.C,all_matches,0,0.20003318786621094,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),first_match,0,0.7081031799316406,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),all_matches,0,0.7081031799316406,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,first_match,0,0.1690387725830078,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,all_matches,0,0.1690387725830078,"[-1, -1]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),first_match,1,0.39577484130859375,"[41351, 41359]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),consecutive,1,0.39577484130859375,"[41351, 41359]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),all_matches,1,0.7586479187011719,"[-1, -1]"
(((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A))(L|I|V|M),first_match,0,0.1499652862548828,"[-1, -1]"
(((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A))(L|I|V|M),all_matches,0,0.1499652862548828,"[-1, -1]"
(A|G....GK)(S|T),first_match,1,0.12087821960449219,"[57, 58]"
(A|G....GK)(S|T),consecutive,1,0.12087821960449219,"[57, 58]"
(A|G....GK)(S|T),consecutive,2,0.012874603271484375,"[94, 95]"
(A|G....GK)(S|T),all_matches,2,0.13375282287597656,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)GT,first_match,0,0.6382465362548828,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)GT,all_matches,0,0.6382465362548828,"[-1, -1]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,first_match,0,0.21696090698242188,"[-1, -1]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,all_matches,0,0.21696090698242188,"[-1, -1]"
